backend/docs/data-model.md

```diff
@@ -0,0 +1,49 @@
+# Data Model
+
+PostgreSQL is used as data persistence layer.
+
+## Environment Variables 
+
+The following environment variables can be used to overwrite the default values:
+
+- `DB_USER`: The user for the database connection (default: `postgres`).
+- `DB_HOST`: The host for the database connection (default: `localhost`).
+- `DB_DATABASE`: The db name for the database connection (default: `postgres`).
+- `DB_PASSWORD`: The password for the database connection (default: `null`).
+- `DB_PORT`: The port for the database connection (default: `null`).
+
+## Tables
+
+All tables are defined in `lib/tables.js`.
+The key in the `lib/tables.js` file matches the table name.
+If the tables don't exist, they are created at server start with the script defined in `lib/tables.js`.
+See the create script for more details about the data types and keys.    
+
+### config
+
+`config` contains settings for the shop like the name, geographical position, etc. 
+
+- `name`: Name of the configuration as a string.
+  At the moment only `default` is used.
+- `data`: The config as a JSON object.
+
+### tickets
+
+`tickets` contains all issued tickets. 
+
+- `id`: Id of the ticket as a string.
+- `user`: User that issued the ticket as a string.
+- `start`: Start of the validity of the ticket as a date object.
+- `end`: End of the validity of the ticket as a date object.
+- `ready`: Boolean flag if the ticket can be used immediately.
```

Comment:
`ready` and `used` can not be true at the same time, I suppose. What do you think about having a `status` field instead of the flags with either `ready` or `used`? Could help to avoid illegal states.

Do we need a `inUse` state as well or would a scanned ticket be `used` directly?

Reply:
At the moment it's not even used. This is more for the future to support counting people in the shop. I'm not sure if there will be a check-in feature in v1.0.0. If not we could remove it completely and collect ideas for the future at a separate place.